Code/README.md

```diff
@@ -1,50 +1 @@
-# Code
-
-This folder will be used to hold all class assigments.
-
-Please adhear to the following folder structure for your labs.
-
-```
-class_kiwi
-│   0 General
-│   1 Python
-│   README.md
-└─  Code
-    │   README.md (this file)
-    └─  harry
-    │   │   README.md
-    │   └─  python
-    │   │       lab1.py
-    │   │       lab2.py
-    │   │       lab3.py  
-    │   └─  html-css
-    │   │       lab1.html 
-    │   │       lab2.html
-    │   └─  javascript
-    │   │       lab1.js
-    │   │       lab2.js
-    └─  ron
-    │   │   README.md
-    │   └─  python
-    │   │       lab1.py
-    │   │       lab2.py
-    │   │       lab3.py  
-    │   └─  html-css
-    │   │       lab1.html 
-    │   │       lab2.html
-    │   └─  javascript
-    │   │       lab1.js
-    │   │       lab2.js
-    └─  hermione
-        │   README.md
-        └─  python
-        │       lab1.py
-        │       lab2.py
-        │       lab3.py  
-        └─  html-css
-        │       lab1.html 
-        │       lab2.html
-        └─  javascript
-        │       lab1.js
-        │       lab2.js
-```
\ No newline at end of file
+# Shaina Davis
```

Comment:
Looks like this alters the original `class_kiwi/Code/README.md`. You will either need to revert it back to its original state or reattempt the lab.